dev/src/v1beta1/firestore_client.js

```diff
@@ -18,9 +18,8 @@ const gapicConfig = require('./firestore_client_config');
 const gax = require('google-gax');
 const merge = require('lodash.merge');
 const path = require('path');
-const pkgUp = require('pkg-up');
 
-const VERSION = require(pkgUp.sync(__dirname)).version;
+const VERSION = require('../../../package').version;
```

Comment:
suggestion: I would recommend using the suffix `.json` even though it is not necessary.

Reply:
Done